bundles/org.openhab.binding.freeboxos/README.md

```diff
@@ -0,0 +1,235 @@
+
+| Parameter Label              | Parameter ID              | Description                                                                                 | Required | Default |
+|------------------------------|----------------------|---------------------------------------------------------------------------------------------|----------|---------|
+| Phone State Refresh Interval | refreshInterval      | The refresh interval in seconds which is used to poll given Freebox Server for phone state. | false    | 2       |
+| Phone Calls Refresh Interval | refreshCallsInterval | The refresh interval in seconds which is used to poll given Freebox Server for phone calls. | false    | 60      |
+
+### Network device / Host
+
+The *host* thing requires the following configuration parameters:
+
+| Parameter Label | Parameter ID | Description                            | Required |
+|-----------------|--------------|----------------------------------------|----------|
+| MAC Address     | macAddress   | The MAC address of the network device. | true     |
+
+### AirPlay device
+
+The *airplay* thing requires the following configuration parameters:
+
+| Parameter Label | Parameter ID | Description                 | Required |
+|-----------------|--------------|-----------------------------|----------|
+| Name            | name         | Name of the AirPlay device  | true     |
+| Password        | password     | AirPlay password            | false    |
+| Accept all MP3  | acceptAllMp3 | Accept any bitrate for MP3 audio or only bitrates greater than 64 kbps | false    |
+
+## Authentication
+
+You will have to authorize openHAB to connect to your Freebox. Here is the process described :
+
+**Step 1** At binding startup, if no token is recorded in the Freebox Server (bridge) configuration, the binding will run a pairing request
+
+
+**Step 2** Run to your Freebox and approve the pairing request for openHAB Freebox Binding that is displayed on the Freebox screen
+
+**Step 3** the application token is automatically recorded in the Freebox Server (bridge) configuration
+
+**Step 4** you can use the console command `freebox <bridgeUID> apptoken` to display the application token and use it later to set up your thing in a configuration file
+
+**Optionally** you can log in your Freebox admin console to allocate needed rights to openHAB
+
+Once initialized, the thing will generate all available channels.
+
+## Channels
+
+The following channels are supported:
+
+| Thing         | Channel Type ID          | Item Type | Access Mode | Description                                                                     |
+|---------------|--------------------------|-----------|-------------|---------------------------------------------------------------------------------|
+| revolution    | fwversion                | String    | R           | Version of the Freebox Server firmware                                          |
```

Comment:
The Channel Type ID should be lowerCamelCase and I would avoid using # and - in it as well.
See https://www.openhab.org/docs/developer/guidelines.html#java-coding-style

Since this is a big change that means not only the thing-type.xml file, source code files, and the readme changed, I wont review it further until this is done so that anyone reviewing can help catch any areas that were missed. I will do a quick speed look over to see if anything else big needs changing.